xapian-letor/tests/api_letor.cc

```diff
@@ -104,26 +98,19 @@ gen_uniqterms_gt_doclen_db(Xapian::WritableDatabase& db, const string&)
     db.add_document(doc);
 }
 
-DEFINE_TESTCASE(ranker1, generated) {
+DEFINE_TESTCASE(listnet_ranker, generated) {
     Xapian::ListNETRanker ranker;
     string a = get_database_path("apitest_ranker1",
-				       gen_uniqterms_gt_doclen_db);
+				       populate_db);
```

Comment:
I am not sure whether I have resolved this ..... what I do generally is that, if a line crosses 80 columns, I simply put enter after the nearest comma